### DISCUSSION TITLE
Lidemy 影片播放 如何自動全螢幕撥放 ?

### DISCUSSION THREAD

**Discussion by kevindtrt** [Lidemy/forum#100] [其他問題] [answered]
現在在學習平台上觀看影片有遇到一個比較大的問題。

平台影片有提供自動完成和自動撥放的選項，所以在看完影片後會自動跳一下個課程。
但如果原先在全螢幕的介面下觀看，自動撥放後卻會跳回瀏覽器的介面。
請問有可能能提供像 youtube 自動撥放時也同時為維持全螢幕的狀態嗎 ?

**Comment by allenliaooooooo** [comment]
可能很多人知道了
不過還是提供一個可以不用滑鼠就可以全螢幕的小撇步
按鍵盤的 F 也可以讓影片全螢幕
當然還是沒有維持全螢幕來的方便🤣
然後 shift + < 或 > 也可以像 YouTube 一樣加速或減速影片播放速度👍

**Comment by aszx87410** [comment, marked as answer]
平台好像不支援這功能QQ
